lib/cdo/shared_constants/pd/facilitator_common_application_constants.rb

```diff
@@ -7,168 +7,148 @@ def self.clean_multiline(string)
+        csf_good_standing_requirement: 'Can you commit to remaining in good standing with Code.org and your assigned Regional Partner?',
+        code_org_facilitator: 'Are you currently (or have you been) a Code.org facilitator?',
+        code_org_facilitator_years: 'In which years did you work as a Code.org facilitator? Mark all that apply.',
+        code_org_facilitator_programs: "Please check the Code.org programs you currently facilitate, or have facilitated in the past. Mark all that apply.",
       },
 
-      section_3_leading_students: {
-        led_cs_extracurriculars: 'Have you led or organized extracurricular computer science learning experiences? Please mark all that apply.',
-        teaching_experience: 'Do you have classroom teaching experience for K-12 students or adults?',
-        grades_taught: 'What grade levels have you taught __in the past?__ Check all that apply.',
-        grades_currently_teaching: 'What grade levels do you __currently__ teach? Check all that apply.',
-        subjects_taught: 'Which subjects do you currently or have you previously taught? Check all that apply.',
-        years_experience: 'How many years of experience do you have teaching computer science for K-12 students or adults?',
-        experience_leading: 'Which of the following do you have experience leading __as a teacher__ (mark all that apply)?',
-        completed_pd: 'Which of the following Code.org professional learning programs did you complete __as a teacher__ (mark all that apply)?'
+      section_3_experience_and_commitments: {
+        teaching_experience: 'Do you have experience as a K-12 classroom teacher?',
+        have_led_adults: 'Have you led learning experiences for adults?',
+        csf_summit_requirement: 'Can you commit to attending the 2019 Facilitator Summit (May 17-19, 2019 in Atlanta, GA)?',
+        csf_workshop_requirement: 'Can you commit to facilitating a minimum of 4-6 one-day workshops starting summer 2019 and continuing throughout the 2019-2020 school year?',
+        csf_community_requirement: 'Can you commit to attending monthly webinars, or watching recordings, and staying up to date through bi-weekly newsletters and online facilitator communities?',
+        csd_csp_fit_weekend_requirement: 'Can you commit to attending one 3-day 2019 Facilitator-in-Training Workshop?',
+        csd_csp_which_fit_weekend: 'Please indicate which FiT Workshop you are able to attend.',
+        csd_csp_workshop_requirement: 'Can you commit to leading 2019-20 academic year workshops hosted by your assigned Regional Partner (generally four days across the academic year)?',
+        csd_training_requirement: 'Can you commit to attending one 5-day training in the spring of 2020 to prepare to co-lead five-day summer workshops?',
+        csp_training_requirement: 'Can you commit to attending __two__ 5-day trainings in the spring of 2020 to prepare to co-lead five-day summer workshops?',
```

Comment:
I should have been more specific in my previous comment. These should go back to saying “__one__ 2- or 3-day training” and “__two__ 2- or 3-day trainings” respectively. I just meant that instead of “five-day summer workshops” (later in the question) we should write it “5-day summer workshops” because that is how we refer to summer workshops everywhere else.